### PR TITLE
Implement TFU-02-03 Remove validators

### PR DESCRIPTION
**Pull request by bailantaotao**

### CODE REVIEW

File: container/blockchain.go [hunk 1]
```diff
@@ -109,6 +111,32 @@ func (bc *blockchain) EnsureConsensusWorking(geths []Ethereum, t time.Duration)
...
+				isFound = true
+			}
+		}
+		if !isFound {
+			newValidators = append(newValidators, v)
+		}
+	}
+
+	// FIXME: It is not good way to wait validator vote out candidates
+	<-time.After(time.Duration(math.Pow(2, float64(len(candidates)))*7) * time.Second)
```
Comment (alanchchen) [inline container/blockchain.go]: How about letting it be a function parameter? The caller can determine the waiting time.

Comment (bailantaotao) [inline container/blockchain.go]: Sure, i've done yet

File: tests/dynamic_test.go [hunk 2]
```diff
@@ -84,4 +85,48 @@ var _ = Describe("Dynamic validators addition/removal testing", func() {
 			Expect(len(validators)).Should(BeNumerically("==", numberOfValidators+testValidators))
 		}
 	})
+
+	It("TFS-02-03 Remove validators", func() {
+		numOfCandidates := 3
+
+		By("Ensure that numbers of validator is equal than $numberOfValidators", func() {
```
Comment (yutelin) [inline tests/dynamic_test.go]: equal to. same type in line 66

File: tests/dynamic_test.go [hunk 3]
```diff
@@ -84,4 +85,48 @@ var _ = Describe("Dynamic validators addition/removal testing", func() {
...
+			Expect(blockchain.EnsureConsensusWorking(blockchain.Validators(), 10*time.Second)).Should(BeNil())
+		})
+
+		for _, v := range blockchain.Validators() {
+			client := v.NewIstanbulClient()
+			validators, err := client.GetValidators(context.Background(), nil)
+			Expect(err).Should(BeNil())
+			Expect(len(validators)).Should(BeNumerically("==", numberOfValidators))
+		}
+		By("Ensure that consensus is working in 10 seconds", func() {
```
Comment (yutelin) [inline tests/dynamic_test.go]: 30 seconds

Comment (bailantaotao) [inline tests/dynamic_test.go]: done

File: tests/dynamic_test.go (hunk 2 above)
Comment (bailantaotao) [inline tests/dynamic_test.go]: done

**Pull request merged**